### ISSUE TITLE
native <Picker />

### ISSUE THREAD

**Issue by ismailkhowaja1** [pixielabs/cavy#83]
How can i trigger this <Picker/> component of react-native. ❓ 

I tried the following: 

`const picker = spec.findComponent('HomeScreen.Picker');
picker.selectedValue="wants";` 

there is no onPress for this component

**Comment by jalada** [comment]
What about triggering the `onValueChange` prop? There's not a spec helper for that, but you should be able to write one yourself.

**Comment by ismailkhowaja1** [comment]
Thank you for your response but we switched it to use codecepjs mobile testing. Although, it was hard to setup initially with appium. 

This framework is nice and very easy to setup but since i am not very familiar with it. I was unable to work with it properly. 

It was giving me errors when i hook components that we wrote. Such as component you are exporting is undefined.

Not every component has onpress or changetext. Due to not enough suport on stack overflow or any where, and not enough documentation to look, i couldn't figure out what i can do. 

We had to change code to wrap app around the cavy which my team was iffy about. 

Good things about this framework is that, it runs super fast, dont need apk, easy to setup and it works flawless when do. 

I hoping to see this framework grow so i can use it my future projects.
